### PR TITLE
Add external projects to srgen

### PR DESCRIPTION
**Pull request by smartguest**
In case any changes are made to the external strings, they will be localized as well now.

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -669,7 +674,11 @@ Task("SRGen")
                 var xlfFile = xlfDoc.Files.Single();
 
                 // load a language template
-                var templateFileLocation = System.IO.Path.Combine(locTemplateDir, System.IO.Path.GetFileName(docName) + ".template");
+                var templateDirToUse = locTemplateDir;
+                if(parentDir.Equals("external")){
```
Comment (Charles-Gagnon) [inline build.cake:678]: Do we actually need this check? The loc folder is the same root folder for both it seems

Comment (smartguest) [reply build.cake:678]: Yeah you're right about that, it's completely unnecessary

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by smartguest** [comment]
Update from Karl, localization to the external folder is not necessary to do so due to the files being archived, this PR will be closed.

**Comment by Charles-Gagnon** [comment]
@kburtram Any issues with just deleting that stuff then at this point? It's been long enough that I don't really see huge value in keeping this around since we don't have any intention in bringing it back (or if we do it'll probably be quite different)

**Pull request closed without merging**